### PR TITLE
Create blaze.md

### PR DESCRIPTION
**Pull request by djjxx1**
## Information

- [ ] This PR fixes a formatting issue.
- [ ] This PR corrects the information in an existing guide.
- [x] This PR adds a new guide.

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] I am sure the guide is formatted properly.

## Licensing

- [x] I agree that my contributions shall be licensed under the [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/),
  and that SkyKings may distribute my contributions under the same license on their website.

### CODE REVIEW

**Review by DaytonWallace — CHANGES_REQUESTED** [review]
Move blaze.md into the guides folder

**Review by M4aXim — CHANGES_REQUESTED** [review]
Please fix Rcommended to Recommended

File: guides/blaze.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+title: The Ultimate Blaze Slayer Guide
+description: |
+  Learn how to get good at blaze slayer.
+category: Bosses
+author: SkyKings
+tags:
+  
+blaze
+inferno
+slayer
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
tags:
  - blaze
  - inferno
  - slayer
```

File: guides/blaze.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+``` 
+So you're struggling with blaze slayer? Well, you have came to the right place! Here you will learn how to defeat all tiers of blaze slayer
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion

So you're struggling with blaze slayer? Well, you have come to the right place! Here you will learn how to defeat all tiers of blaze slayer
```

File: guides/blaze.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+bosses, places to spawn and cheese the boss and extra info about blaze slayer. Now, lets get into it.
+
+Part 1: Gear Requirements
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
## Part 1: Gear Requirements
```

File: guides/blaze.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+Tier 1
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
### Tier 1
```

File: guides/blaze.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+Sword: Livid Dagger
+
+Armour: Full Superior / Shadow Assassin
+
+Pet(s): Rare Lion (shelmet is reccomended)
+
+Tips: Not really much to say, just the basic tier of blaze slayer.
+
+Tier 2
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
### Tier 2
```

File: guides/blaze.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+Sword: Firedust Dagger, Twilight Dagger
+
+Armour: Full Hot Crimson
+
+Pet(s): Epic Lion, Epic Tiger
+
+Tips: If you can't survive, try putting 1 piece of sorrow on.
+
+Tier 3
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
### Tier 3
```

File: guides/blaze.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+Sword: Kindlebane Dagger, Mawdredge Dagger
+
+Armour: Full Hot Crimson 10✪
+
+Pet(s): Legedary Griffin (Reccomended), Wolf/Wisp (Spawning)
+
+Tips: If you can't survive, try putting on a sorrow chestplate
+
+Tier 4
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
### Tier 4
```

File: guides/blaze.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+Withercloak (for pillars), Wisp Potion (if you're dying), Florid Zombie Sword
+
+
+Tips for all tiers: Try to have shelmet as your pet item as it helps out a lot during the boss. Make sure you have vitality on your crimson or 
+else you won't be able to survive. If you have a warden helmet and the less vitality doesn't really effect you, try using it for more dps. You
+should also be using either full crimson equipment set or full vanquished equipment set for all tiers (excluding tier 1) for the extra damage and 
+damage reduction. Try to have vitality on your equipment for the extra healing. For the second attributes, try using dominance for extra damage
+or veteran for faster spawning.
+
+Part 2: Places to spawn and cheese
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
## Part 2: Places to spawn and cheese
```

File: guides/blaze.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+Tips: If you can't survive, try putting on a sorrow chestplate
+
+Tier 4
+
+Sword: Kindlebane Dagger, Mawdredge Dagger
+
+Armour: 3/4 Burning Crimson 5✪, Sorrow Chestplate
+
+Pet(s): Legendary Griffin (Cheep), Legendary Enderdragon (Reccomended),  Wolf/Wisp (Spawning)
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
Pet(s): Legendary Griffin (Cheap), Legendary Enderdragon (Recommended), Wolf/Wisp (Spawning)
```

File: guides/blaze.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+Tips: If you can't survive, try putting 1 piece of sorrow on.
+
+Tier 3
+
+Sword: Kindlebane Dagger, Mawdredge Dagger
+
+Armour: Full Hot Crimson 10✪
+
+Pet(s): Legedary Griffin (Reccomended), Wolf/Wisp (Spawning)
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
Pet(s): Legendary Griffin (Recommended), Wolf/Wisp (Spawning)
```

File: guides/blaze.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+Part 1: Gear Requirements
+
+Tier 1
+
+Sword: Livid Dagger
+
+Armour: Full Superior / Shadow Assassin
+
+Pet(s): Rare Lion (shelmet is reccomended)
```
Comment (plun1331) [inline guides/blaze.md]:
```suggestion
Pet(s): Rare Lion (shelmet is recommended)
```

**Review by plun1331 — CHANGES_REQUESTED** [review]

**Review by plun1331 — DISMISSED** [review]

**Review by DaytonWallace — APPROVED** [review]

**Review by plun1331 — APPROVED** [review]

**Pull request merged**